.env.example

```
@@ -65,6 +65,8 @@ CALCOM_TELEMETRY_DISABLED=
 
 # ApiKey for cronjobs
 CRON_API_KEY='0cc0e6c35519bba620c9360cfe3e68d0'
+# Secret used by tasker/cron endpoints (Authorization: Bearer <CRON_SECRET>). Must be set; if unset the endpoint rejects all requests.
+CRON_SECRET=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)